test/utils/base_page/base_page.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../../../test/config/logger.config.js');
+const Header = require('./header');
+
+class BasePage {
+    constructor() {
+        this.Header = new Header();
+    };
+    wait(seconds) {
+        logger.info(`Waiting "${seconds * 1000}" milliseconds`);
+        return browser.sleep(seconds * 1000);
+    };
+    async getCurrenUrl() {
+        const currentUrl = browser.getCurrentUrl();
+        logger.debug(`Current url is "${currentUrl}"`);
+        return currentUrl;
+    };
+    open(url) {
```

Comment:
In this case it is better to do on e of these 2 options:
1) Add `this.url` to your constructor which will be your page URL and than use it in this method:
```
open() {
   logger.info(`Opening "${this.url}" url`);
   browser.get(this.url);
}
```
2) Move the whole method to a separate function helpter:
```
async function openPage(url) {
   logger.info(`Opening "${url}" url`);
   browser.get(url);
}

Reply:
done